grant-project/zcash-grant-system
# Various Fixes

Closes #262 
Closes https://github.com/grant-project/zcash-grant-system/issues/263
Closes https://github.com/grant-project/zcash-grant-system/issues/229
Closes https://github.com/grant-project/zcash-grant-system/issues/222
Closes https://github.com/grant-project/zcash-grant-system/issues/216


#### What this does
Hits a variety of low-hanging fruit. 

### To test:
- #262: Make sure popover is no longer available when clicking cancel & refund when button is disabled
- #263 Make sure "delete" is no longer shown on the proposal listview
- #229 Attempt to post a comment longer than 1000 chars
- #222 Attempt to commit in `admin` with an unused import or otherwise fail `tsc`. Interestingly, we did already run `tsc` before `push`es, but since it was so troubling to have the heroku deployment fail, I figured we could also add it to pre-commits.
- https://github.com/grant-project/zcash-grant-system/issues/216 Trigger an email to send. Confirm a real address is shown in the footer.
